plugins/hw_wallet/plugin.py

```diff
@@ -51,3 +51,13 @@ def close_wallet(self, wallet):
         for keystore in wallet.get_keystores():
             if isinstance(keystore, self.keystore_class):
                 self.device_manager().unpair_xpub(keystore.xpub)
+
+def is_any_tx_output_on_change_branch(tx: Transaction):
+    if not tx.output_info:
```

Comment:
Hmm.  It probably doesn't matter for the extant code you have here -- but to reduce the burden on future programmers, and not raise AttributeError here  -- would you mind doing a check for the attribute here, eg `if not getattr(tx, 'output_info', None):` instead?  

It seems `tx.output_info` isn't always defined for every Transaction instance (blame this codebase).  Just defensive programming...

Reply:
(And yes, I realize in this callpath it will only ever be reached after `wallet.add_hw_info` runs.. :)

Reply:
AFAICT it is defined for any TX for a hardware wallet in `add_hw_info`. I'll add the check though.